extension/view/js/translation-notification-fxtranslations.js

```diff
@@ -200,57 +151,13 @@ window.MozTranslationNotification = class extends MozElements.Notification {
     this.state = this.translationNotificationManager.TranslationInfoBarStates.STATE_TRANSLATING;
   }
 
-  _close() {
-
-    this.close();
-
-    /*
-     * todo: this.translation is undefined
-     * const from = this._getSourceLang();
-     * const to = this._getTargetLang();
-     * this.translation.infobarClosed(from, to);
-     */
-  }
-
   /*
    * to be called when the infobar should be closed per user's wish (e.g.
    * by clicking the notification's close button, the not now button or choosing never to translate)
    */
   closeCommand() {
     this.translationNotificationManager.reportInfobarEvent("closed");
-    this._close();
-  }
-
-  /*
-   * to be called when the infobar should be closed per user's wish
-   * by clicking the Not now button
-   */
-  notNow() {
-    this.translationNotificationManager.reportInfobarEvent("not_now");
-    this._close();
-  }
-
-  _handleButtonHiding() {
-    const originalShown = this.translation.uiState.originalShown;
-    this._getAnonElt("showOriginal").hidden = originalShown;
-    this._getAnonElt("showTranslation").hidden = !originalShown;
-  }
-
-  showOriginal() {
-    this.translationNotificationManager.reportInfobarEvent("accept_outbound");
```

Comment:
what about this one?

Reply:
So this event being here was already a mistake. Not only because this function was never called since the panel which contains the element that calls it is never displayed and is disabled, but also its purpose (if you look at the where it is invoked) is to revert from the translated state to the original state of the page, and has nothing to do with what the `accept_outbound` event entails. It seems a lot of these events were wrongly attached to dead and disabled code already that's never being invoked.